docs/getting_started/validation.md

```diff
@@ -44,11 +44,12 @@ positional arguments:
   base_url              The base URL of the OPTIMADE
                         implementation to point at, e.g.
                         'http://example.com/optimade/v1' or
-                        'http://localhost:5000/v1
+                        'http://localhost:5000/v1'
 
 optional arguments:
-  -h, --help            show this help message and exit
+  -h, --help            Show this help message and exit.
   -v, --verbosity       Increase the verbosity of the output.
+                        (-v: warning, -vv: info, -vvv: debug)
```

Comment:
I believe this section is copied straight from the output in a terminal when running the `--help` command. Are these updates from the same? If not, it might be good to edit also the specific places in the code to fix these things.

Reply:
The original text was indeed from `--help`. I updated the text for the help response. The '--help' line is however the default line from argparse. I can override this response, but it would take a few extra lines of code, I am not sure if the extra capital is worth it.